data/json/items/generic/filament.json

```diff
@@ -0,0 +1,67 @@
+[
+  {
+    "id": "thread",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": "thread",
+    "description": "A small quantity of thread that could be used when sewing.",
+    "weight": 1,
+    "volume": "1ml",
+    "stackable": true,
+    "price": 5,
+    "material": "cotton",
+    "symbol": "=",
+    "color": "light_gray",
+    "count": 50,
+    "stack_size": 200,
+    "flags": [ "NO_SALVAGE" ]
+  },
+  {
+    "id": "sinew",
```

Comment:
Those 3 filaments below differ from each other by name, description, material and color.
You should define a base filament and have all of them `copy-from` it instead of redefining everything anew. Check out how it's done for car batteries.